Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/UnityProjectInfo.cs

```diff
@@ -213,6 +221,10 @@ public void RefreshProjects(MSBuildToolsConfig config)
                 }
             }
 
+            // Now we have all of the assembly definiton files, let's run a quick validation on the references to correct known errors like "unity.ugui" supposed to be "UnityEngine.UI"
+            /// <see cref="ProjectAliases"/> for the correction map.
```

Comment:
NIT on the comment style.  Double // on one line and triple on the other?

Reply:
Yeah, just to put the <see cref; but on further thought I will just move it to the function


---
In reply to: [365450151](https://github.com/microsoft/MSBuildForUnity/pull/102#discussion_r365450151) [](ancestors = 365450151)